core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -30,6 +30,7 @@ class BaseUnit : INamed, IConstruction, CivilopediaText() {
     var interceptRange = 0
     lateinit var unitType: UnitType
     var requiredTech: String? = null
+    var requiredPolicy: String? = null
```

Comment:
No. This should be by unique, since very few buildings have this. Unlike required tech, which is almost every building.

Reply:
If you want to make a mod, that will be good. I think splitting construction requirement and building effect is a good idea.